.github/workflows/sims-cron.yml

```diff
@@ -0,0 +1,70 @@
+name: Sims Cron Job
+
+on:
+  schedule:
+    # Runs "at 1am every day"
+    - corn: "0 0 * * *"
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: "!contains(github.event.head_commit.message, 'skip-sims')"
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2.1.3
+        with:
+          go-version: 1.18
+      - name: Display go version
+        run: go version
+      - run: make build
+
+  install-runsim:
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - uses: actions/setup-go@v2.1.3
+        with:
+          go-version: 1.18
+      - name: Display go version
+        run: go version
+      - name: Install runsim
+        run: export GO111MODULE="on" && go install github.com/cosmos/tools/cmd/runsim@v1.0.0
+      - uses: actions/cache@v2.1.3
+        with:
+          path: ~/go/bin
+          key: ${{ runner.os }}-go-runsim-binary
+
+sim-after-import:
+    runs-on: ubuntu-latest
+    needs: [build, install-runsim]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2.1.3
+        with:
+          go-version: 1.18
+      - name: Display go version
+        run: go version
+      - uses: actions/cache@v2.1.3
+        with:
+          path: ~/go/bin
+          key: ${{ runner.os }}-go-runsim-binary
+      - name: sim-after-import
+        run: |
+          make sim-regen-after-import
+
+  sim-fullapp:
+    runs-on: ubuntu-latest
+    needs: [build, install-runsim]
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2.1.3
+        with:
+          go-version: 1.18
+      - name: Display go version
+        run: go version
+      - uses: actions/cache@v2.1.3
+        with:
+          path: ~/go/bin
+          key: ${{ runner.os }}-go-runsim-binary
+      - name: sim-fullapp
+        run: |
```

Comment:
Maybe we should add a discord alert for these actions.

Reply:
I'll look into this and see if we can get that set up. :+1: